test/browser/browser_topsites_section.js

```diff
@@ -116,3 +118,21 @@ test_newtab({
     await ContentTaskUtils.waitForCondition(() => (content.document.querySelector(".top-site-outer:first-child a").getAttribute("href") !== "https://bugzilla.mozilla.org"), "Topsite not removed");
   }
 });
+
+test_newtab({
+  before: setDefaultTopSites,
+  test: async function test_search_topsite_keyword() {
+    await ContentTaskUtils.waitForCondition(() => content.document.querySelector(".search-shortcut .title.pinned"), "Wait for pinned search topsites");
+
+    const searchTopSites = content.document.querySelectorAll(".title.pinned");
+    ok(searchTopSites.length >= 2, "There should be at least 2 search topsites");
+
+    searchTopSites[0].click();
+
+    return searchTopSites[0].innerText;
+  },
+  after: searchTopSiteTag => {
+    ok(gURLBar.focused, "We clicked a search topsite the focus should be in location bar");
+    ok(gURLBar.value.includes(searchTopSiteTag), "Should contain the tag of the search topsite clicked");
```

Comment:
Not sure if we need to clean up the value as tests open up new tabs anyway and this one is going away. Also, unclear if we need to wait any longer than the content task finishing.

I suppose if tests and verify are happy, then no need to change.